src/features/collapsible-sidebar/__tests__/CollapsibleSidebar.test.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
Nit: I think we're trying to move away from sinon. Could you get by with jest style mocks instead?